django_sendfile/utils.py

```diff
@@ -34,7 +34,11 @@ def _convert_file_to_url(path):
     relpath = path_obj.relative_to(path_root)
     # Python 3.5: Path.resolve() has no `strict` kwarg, so use pathmod from an
     # already instantiated Path object
-    url = relpath._flavour.pathmod.normpath(str(url_root / relpath))
+    if hasattr(relpath._flavour, "pathmod"):
+        url = relpath._flavour.pathmod.normpath(str(url_root / relpath))
+    else:
+         # Python 3.12 doesn't expose pathmod anymore
+        url = relpath._flavour.normpath(str(url_root / relpath))
```

Comment:
Why don't we just use `Path.resolve`? The lowest version of Python we support is 3.7, so 3.5 is no longer a concern.

Reply:
Tried it myself, I guess that comment is a little misleading as it doesn't work on `PurePath` objects